### PR TITLE
Update analytics

### PR DESCRIPTION
**Pull request by pinzart90**
Remove all usage of IsOptedIn for ADP (it is considered an internal API by ADP)
Added `USEEXCEPTIONTRACE` option for CER as per this discussion https://autodesk.slack.com/archives/CH8CL3MRR/p1664815775433379

### CODE REVIEW

File: src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs [hunk 1]
```diff
@@ -474,7 +475,7 @@ private Step CreateStep(Step jsonStepInfo, HostControlInfo hostControlInfo, int
                         Sequence = jsonStepInfo.Sequence,
                         RatingTextTitle = formattedText.ToString(),
                         StepType = Step.StepTypes.SURVEY,
-                        IsRatingVisible = dynamoViewModel.Model.PreferenceSettings.IsADPAnalyticsReportingApproved,
```
Comment (pinzart90) [inline src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs:477]: @mjkkirschner @QilongTang any ideas what this is used for ?

Comment (QilongTang) [reply src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs:477]: This is about the five star rating after guided tour exited, when analytics opt-in, we display the rating for ppl to rate

Comment (pinzart90) [reply src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs:477]: well now we will no longer know if ADP is opted in or not.
Should I just check for google analytics ? or as it is (if no one explicitly disabled analytics) ?

Comment (QilongTang) [reply src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs:477]: After this change, it seems only startup params would impact which I think should apply. I would leave it as it is. If user opt-out for GA, I think the client would send nothing right?

File: src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs [hunk 2]
```diff
@@ -474,7 +475,7 @@ private Step CreateStep(Step jsonStepInfo, HostControlInfo hostControlInfo, int
...
+                        IsRatingVisible = !Analytics.DisableAnalytics && Analytics.IsEnabled,
```
Comment (mjkkirschner) [inline src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs:478]: will `Analytics.IsEnabled` always be true?

File: src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs (hunk 1 above)
Comment (pinzart90) [reply src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs:477]: Sounds right

File: src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs (hunk 2 above)
Comment (pinzart90) [reply src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs:478]: IsEnabled is false when the DynamoAnalyticsClient is not created. DynamoAnalyticsClient is created when dynamo model is started with these flags
```
!IsServiceMode && !areAnalyticsDisabledFromConfig && !Dynamo.Logging.Analytics.DisableAnalytics
```

Comment (pinzart90) [reply src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs:478]: There might be some other conditions depending on how the Dynamo model is created

**Pull request merged**